### PR TITLE
Update `ArgumentOutOfRangeException.ThrowIfNullOrWhiteSpace` usage in `FileUtil`

### PR DESCRIPTION
**Pull request by WeihanLi**
use `Debug.Assert` instead

### CODE REVIEW

**Comment by danmoseley** [comment]
This is strictly speaking a breaking change (not that it matters much in Preview), but it does seem like an improvement.

File: src/Aspire.Hosting/Utils/FileUtil.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@ internal static class FileUtil
 {
     public static string FindFullPathFromPath(string command)
     {
-        ArgumentOutOfRangeException.ThrowIfNullOrWhiteSpace(command, nameof(command));
+        ArgumentException.ThrowIfNullOrWhiteSpace(command);
```
Comment (eerhardt) [inline src/Aspire.Hosting/Utils/FileUtil.cs:10]: This is an internal API that is only called with constant strings. We should just turn this into a `Debug.Assert`.

**Review by eerhardt — APPROVED** [review]
Thanks for the contribution!

**Comment by danmoseley** [comment]
> Since the type is internal, think it would not break the usage,

It was too early in the morning, I missed that. Thanks

**Comment by WeihanLi** [comment]
Never mind, thanks very much

**Pull request merged**